lib/screenplay_web/controllers/config_controller.ex

```diff
@@ -38,6 +41,21 @@ defmodule ScreenplayWeb.ConfigController do
   def existing_screens(conn, %{"place_ids" => place_ids, "app_id" => app_id}) do
     app_id_atom = String.to_existing_atom(app_id)
 
+    {pending_screens_config, version_id} =
+      case PendingScreensConfig.fetch_config() do
+        {:ok, config, version_id} ->
+          %PendingConfig{screens: pending_screens} =
+            config
+            |> Jason.decode!()
+            |> PendingConfig.from_json()
+
+          {pending_screens, version_id}
+
+        _ ->
+          Logger.error("Could not fetch pending screens config in existing_screens/2")
+          []
```

Comment:
The empty list returned here will cause the server to give a 500 status since `[]` won't match the pattern `{pending_screens_config, version_id}`.

Do we want that? We don't really need to do any kind of graceful handling here since a failed S3 fetch would probably mean something is very broken, but the error that would get raised if this happened could be a bit confusing.

I guess I'm wondering if it would make sense to just `raise` a runtime error instead of / in addition to the log, rather than returning anything from the case expression.

Reply:
Raising works for now, and for the error handling task we could adjust this to send a response with a more informative message for the client to display.

I think this would fall in the bucket of failure modes where we'd tell the user to try again and contact us if it continues to not work.